src/test/java/BadVDiffTest.java

```diff
@@ -0,0 +1,43 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.tinylog.Logger;
+import org.variantsync.diffdetective.diff.result.DiffParseException;
+import org.variantsync.diffdetective.util.StringUtils;
+import org.variantsync.diffdetective.variation.diff.DiffTree;
+import org.variantsync.diffdetective.variation.diff.bad.BadVDiff;
+import org.variantsync.diffdetective.variation.diff.graph.GraphView;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class BadVDiffTest {
+    private static final Path resDir = Constants.RESOURCE_DIR.resolve("bad");
```

Comment:
The directory name `bad` suggests broken test cases. Maybe use `badvdiff` instead?

Reply:
will do